make instructions on README have pandoc dependency installation requirement on macOS 10.14

### Expected behaviour

Use of `make` according to [installation instructions on repo README](https://github.com/mrtazz/checkmake#installation) will complete without errors on macOS v10.14.1 where pandoc is not installed by default.

### Actual behaviour

```
github.com/mrtazz/checkmake  master ✔                                    151d
▶ make
install -d .d
echo "checkmake: $(go list -f '{{ join .Deps "\n" }}' cmd/checkmake/main.go | awk '/github/ { gsub(/^github.com\/[a-z]*\/[a-z]*\//, ""); printf $0"/*.go " }')" > .d/checkmake.d
go build -ldflags "-X 'main.version=0.1.0-14-gff21564' -X 'main.buildTime=2018-11-30T16:50:39Z' -X 'main.builder=Chris Simpkins <git.simpkins@gmail.com>' -X 'main.goversion=go version go1.11.2 darwin/amd64'" -o checkmake cmd/checkmake/main.go
sed "s/REPLACE_DATE/November 30, 2018/" man/man1/checkmake.1.md | pandoc -s -t man -o checkmake.1
/bin/sh: pandoc: command not found
make: *** [checkmake.1] Error 127
```

### Output of `checkmake --version`

```
checkmake  built at  by  with
```

`go get` executed on commit https://github.com/mrtazz/checkmake/commit/ff21564a952cb62b4632a0ba7637499bb8f775fc

### Output of `make --version`

GNU Make 3.81


When pandoc installed with `brew install pandoc` the make execution completes with exit status code 0; however, the version information is missing the build date/time and builder:

```
github.com/mrtazz/checkmake  master ✔                                    151d
▶ checkmake --version
checkmake  built at  by  with
```

--- a/README.md
+++ b/README.md
@@ -45,9 +45,14 @@ Options:
 
 ## Installation
 
-There are packages for linux up [on
-packagecloud.io](https://packagecloud.io/mrtazz/checkmake) but you can also
-just clone the repo and build the binary and man page yourself.
+### Requirements
+The [pandoc](https://pandoc.org/) document converter utility is required to run checkmate. You can find out if you have it via `which pandoc`. [Install pandoc](https://pandoc.org/installing.html) if the command was not found.
+
+### Packages
+There are packages for linux up [on packagecloud.io](https://packagecloud.io/mrtazz/checkmake) or build it yourself with the steps below.
+
+### Build
+To build checkmake you will need to have [golang](https://golang.org/) installed. Once you have Go installed, you can simply clone the repo and build the binary and man page yourself with the following commands.
 
 ```
 go get github.com/mrtazz/checkmake

